microsoft/PowerToys
# [PowerRename] Sync settings before saving last window size

## Summary of the Pull Request


## PR Checklist

- [x] **Closes:** #28854


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
- change settings while PowerRename window is open
- close PowerRename window 
- switch back and forth to PowerRename settings page and verify that they're no longer reverted
